code/game/objects/items/AI_modules.dm

```diff
@@ -68,9 +69,23 @@ AI MODULES
 	var/time = time2text(world.realtime,"hh:mm:ss")
 	var/ainame = law_datum.owner ? law_datum.owner.name : "empty AI core"
 	var/aikey = law_datum.owner ? law_datum.owner.ckey : "null"
-	GLOB.lawchanges.Add("[time] <B>:</B> [user.name]([user.key]) used [src.name] on [ainame]([aikey]).[law2log ? " The law specified [law2log]" : ""]")
-	log_law("[user.key]/[user.name] used [src.name] on [aikey]/([ainame]) from [AREACOORD(user)].[law2log ? " The law specified [law2log]" : ""]")
-	message_admins("[ADMIN_LOOKUPFLW(user)] used [src.name] on [ADMIN_LOOKUPFLW(law_datum.owner)] from [AREACOORD(user)].[law2log ? " The law specified [law2log]" : ""]")
+
+	//affected cyborgs are cyborgs linked to the AI with lawsync enabled
+	var/affected_cyborgs = list()
+	var/list/borg_txt = list()
+	var/list/borg_flw = list()
+	if(isAI(law_datum.owner))
+		var/mob/living/silicon/ai/owner = law_datum.owner
+		for(var/mob/living/silicon/robot/owned_borg in owner.connected_robots)
```

Comment:
```suggestion
		for(var/mob/living/silicon/robot/owned_borg as anything in owner.connected_robots)
```
Can there be a type other than this one in that list?
If not, then there's no need for a type check.